### PR TITLE
Update issue/feature templates

### PR DESCRIPTION
**Pull request by yadudoc**

### CODE REVIEW

**Comment by yadudoc** [comment]
Tagging @dgasmith for comments since he recommended we add templates for issues features.

**Comment by dgasmith** [comment]
You can find ours [here](https://github.com/MolSSI/QCFractal/tree/master/.github/ISSUE_TEMPLATE). A nice [example](https://github.com/MolSSI/QCFractal/issues/414) as well.

**Comment by dgasmith** [comment]
Whoops, thought this was an issue! Reading now.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1. Setup Parsl 0.8.0 with Python 3.6 on cluster
+2. Run a test script
+3. Wait 5 mins
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment**
+ - OS: [e.g. iOS]
```
Comment (ZhuozhaoLi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Perhaps the example should be `ubuntu`, `centos`, `MacOS`, etc.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: yadudoc
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (ZhuozhaoLi) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Ex. -> E.g., ?

**Pull request merged**